internal/interpreter/interpreter_test.go

```diff
@@ -2218,6 +2252,67 @@ func TestVariableBalance(t *testing.T) {
 // 	test(t, tc)
 // }
 
+func TestBalanceSimple(t *testing.T) {
+	script := `
```

Comment:
add t.Parallel() call on your tests except if it's not parallelizable, but I guess it is the case here

Reply:
oh yeah that's true.. yeah I think so far 100% of the tests are parallelizable, but I keep forgetting about adding it.. is there a way to declare that the whole module is parallelizable ?